shipyard/README.md

```diff
@@ -0,0 +1,62 @@
+# Shipyard
+
+## Overview
+
+Shipyard enables teams by providing on-demand (ephemeral) environments for every code change (PR). Shipyard works in your existing CI workflow, empowering teams to test before they merge.
+
+The Shipyard Datadog integration automatically passes your [Shipyard][1] ephemeral environment logs to Datadog for all applications on which you have enabled Datadog logging.
+
+This is useful for: 
+
+- Monitoring your application's runtime status
+- Using Datadog to enrich and parse your runtime logs
+- Uncovering trends and comparing the logs from your application's environments to the rest of your Datadog data
+
+## Setup
+
+**Note:** These instructions assume you have a Shipyard account. If you do not, you can get started [for free][2].
+
+### Configuration
+
+1. Navigate to the [API Keys page][5] and create an API key.
+2. In your Shipyard dashboard, go to **Settings**. 
+3. Under the **Datadog** section, enter your Datadog API key and the Datadog site URL.
+   ![datadog-input][6]
+   1. If you do not see a Datadog section here, email [Shipyard support][3] to enable this feature for your organization.
+   2. To get your Datadog site URL, see [Getting Started with Datadog Sites][4].
+4. Click **Install Datadog**.
+   1. *(Optional)* Toggle **Enable Datadog logging for all environments** if you would like Shipyard to automatically send logs for **all** of your environments.
+5. To enable Datadog logging for select environments, see the [Configure Application page][10]. 
+   1. Select the **Notification and General Settings** tab. 
+   2. Toggle **Enable Datadog logging for this environment**.
+
+For more information about setting up the Shipyard Datadog integration, see the [Shipyard documentation][7]. 
+
+## Data Collected
+
+### Metrics
+
+Shipyard does not include any metrics.
+
+### Service Checks
+
+Shipyard does not include any service checks.
+
+### Events
+
+Shipyard does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Shipyard support][3].
+
+[1]: https://shipyard.build/
+[2]: https://shipyard.build/signup
+[3]: mailto:support@shipyard.build
+[4]: https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site
+[5]: https://app.datadoghq.com/organization-settings/api-keys
+[6]: https://raw.githubusercontent.com/mesmith027/DataDog-integrations-extras/shipyard-integration/shipyard/images/datadog-input.png
```

Comment:
non blocking nit:
The link here should ideally be updated later to integrations extras in case the mesmith027 fork ever gets deleted. 

```
All images must be checked into the repo under the `shipyard/images` folder. This image path must be in the form: https://raw.githubusercontent.com/DataDog/integrations-extras/master/shipyard/images/<IMAGE_NAME> Image currently is: https://raw.githubusercontent.com/mesmith027/DataDog-integrations-extras/shipyard-integration/shipyard/images/datadog-input.png
```

Reply:
Good catch, thank you! Just updated this.